cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -614,6 +617,65 @@ func (s *Server) handleHTTPRoot(w http.ResponseWriter, r *http.Request) {
 	_ = repoTmpl.Execute(w, data)
 }
 
+func (s *Server) handleDebugList(w http.ResponseWriter, r *http.Request) {
+	withIndexed := false
+	indexedStr := r.URL.Query().Get("indexed")
+	if b, err := strconv.ParseBool(indexedStr); err != nil {
+		withIndexed = b
+	}
```

Comment:
if you are ignoring the error (which seems fine) then this works since it will return false on failure:

```suggestion
	withIndexed, _ := strconv.ParseBool(r.URL.Query().Get("indexed"))
```